spec/controllers/setup_controller_spec.rb

```diff
@@ -243,25 +238,14 @@
       end
 
       it "warns and redirects to the setup_path" do
-        put :configure, settings: settings_params.each { |k, _v| settings_params[k] = "" }
+        put :configure, settings: Hash[settings_params.map { |k, _| [k, ""] }]
```

Comment:
there's another line down below for json formatted requests that needs to be adapted as well